src/test/predict_validation_test.cpp

```diff
@@ -165,6 +165,32 @@ TEST_F(PredictValidation, RequestWrongBatchSizeAuto) {
     EXPECT_EQ(status, ovms::StatusCode::BATCHSIZE_CHANGE_REQUIRED);
 }
 
+TEST_F(PredictValidation, RequestWrongBatchSizeAutoBinaryInputs) {
+    modelConfig.setBatchingParams("auto");
+    tensorflow::serving::PredictRequest binaryInputRequest = preparePredictRequest(
+        {
+            {"Binary_Input",
+                std::tuple<ovms::shape_t, tensorflow::DataType>{{1, 224, 224, 3}, tensorflow::DataType::DT_STRING}},
+        });
+    auto& input = (*binaryInputRequest.mutable_inputs())["Binary_Input"];
+    std::string val1 = "val1";
+    std::string val2 = "val2";
+    input.add_string_val(val1);
+    input.add_string_val(val2);
+
+    networkInputs.clear();
+    std::string inputName = "Binary_Input";
```

Comment:
If you declared it at the beginning of a test, you could use it in line 175 aswell